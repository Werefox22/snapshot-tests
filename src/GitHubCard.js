import React from "react";
import Card from 'react-bootstrap/Card'

function GitHubCard() {
	return (
		<Card>
			<Card.Title>Jack Wright</Card.Title>
		</Card>
	)
}

export default GitHubCard