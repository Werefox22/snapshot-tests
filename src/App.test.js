import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'
import GitHubCard from './GitHubCard';

test('renders a snapshot', () => {
	const tree = renderer.create(<App />).toJSON()
	expect(tree).toMatchSnapshot()
})

test('renders a snapshot of the github card', () => {
	const tree = renderer.create(<GitHubCard />).toJSON()
	expect(tree).toMatchSnapshot()
})