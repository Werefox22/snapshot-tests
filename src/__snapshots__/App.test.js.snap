// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="App"
>
  <div
    className="card"
    style={
      Object {
        "margin": "auto",
        "width": "300px",
      }
    }
  >
    <div
      className="card-body"
      style={
        Object {
          "textAlign": "left",
        }
      }
    >
      <img
        alt="Jack Wright Profile Picture"
        className="rounded-circle img-thumbnail"
        src="pfp.jpg"
        width="50px"
      />
      <div
        className="card-title h5"
        style={
          Object {
            "display": "inline",
          }
        }
      >
         Jack Wright
      </div>
      <p
        className="card-text"
      >
        Software development student based in NC.
      </p>
    </div>
  </div>
</div>
`;

exports[`renders a snapshot of the github card 1`] = `
<div
  className="card"
  style={
    Object {
      "margin": "auto",
      "width": "300px",
    }
  }
>
  <div
    className="card-body"
    style={
      Object {
        "textAlign": "left",
      }
    }
  >
    <img
      alt="Jack Wright Profile Picture"
      className="rounded-circle img-thumbnail"
      src="pfp.jpg"
      width="50px"
    />
    <div
      className="card-title h5"
      style={
        Object {
          "display": "inline",
        }
      }
    >
       Jack Wright
    </div>
    <p
      className="card-text"
    >
      Software development student based in NC.
    </p>
  </div>
</div>
`;
